lessons/ro/chapter_1.yaml

```
@@ -13,6 +13,26 @@
 
     Odată ce vă veți familiariza cu Rust, puteți să vă declarați un **Rustaceu**. Așa
     se numesc toți oamenii care folosesc, contribuie sau sunt interesați de Rust.
+
+- title: What is Rust?
+  content_markdown: >
+    Rust este un limbaj de programare al sistemelor care este cunoscut pentru accentul pe care îl pune
+    siguranță, performanță și concurență.
+    
+    Rust este conceput pentru a oferi un control la nivel scăzut asupra resurselor sistemului
+    fără a sacrifica abstracţiile de nivel înalt.
+    
+    Ce are Rust diferit de alte limbaje de programare.
+    - proprietate și împrumut
+    - durata de viață
+    - potrivire de model
```

Since it is a technical term and it is used as such in the code and documentation, it would be useful to also include the english version for more clarity:
```suggestion
    - potrivire de model (pattern matching)
```